### PR TITLE
Replace `CUDA C++ Core Libraries` with `CUDA Core Compute Libraries` (only in README.md).

### PR DESCRIPTION
**Pull request by rwgk**
## Description

Trivial modernization, mainly to practice creating a cccl PR.

## Checklist

- [X] New or existing tests cover these changes.
- [X] The documentation is up to date with these changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,9 +3,9 @@
 |[Contributor Guide](https://github.com/NVIDIA/cccl/blob/main/CONTRIBUTING.md)|[Dev Containers](https://github.com/NVIDIA/cccl/blob/main/.devcontainer/README.md)|[Discord](https://discord.gg/nvidiadeveloper)|[Godbolt](https://godbolt.org/z/x4G73af9a)|[GitHub Project](https://github.com/orgs/NVIDIA/projects/6)|[Documentation](https://nvidia.github.io/cccl)|
 |-|-|-|-|-|-|
 
-# CUDA C++ Core Libraries (CCCL)
+# CUDA Core Compute Libraries (CCCL)
 
-Welcome to the CUDA C++ Core Libraries (CCCL) where our mission is to make CUDA C++ more delightful.
+Welcome to the CUDA Core Compute Libraries (CCCL) where our mission is to make CUDA C++ more delightful.
```
Comment (miscco) [inline README.md:8]: Should this also drop the C++
```suggestion
Welcome to the CUDA Core Compute Libraries (CCCL) where our mission is to make CUDA more delightful.
```

**Review by miscco — APPROVED** [review]
Congrats to the first PR 🎉

File: README.md (hunk 1 above)
Comment (rwgk) [reply README.md]: Thanks! I looked through systematically and found another one: f12ae0f762ecb6a73b8fc25b237adbcece1dc5fd

**Review by gevtushenko — APPROVED** [review]

**Review by bdice — APPROVED** [review]
Thanks!

**Pull request merged**